src/Analyzers/CSharp/CodeFixes/MisplacedUsingDirectives/MisplacedUsingDirectivesCodeFixProvider.cs

```diff
@@ -423,7 +423,7 @@ private static CompilationUnitSyntax AddFileHeader(CompilationUnitSyntax compila
         private class MoveMisplacedUsingsCodeAction : CustomCodeActions.DocumentChangeAction
         {
             public MoveMisplacedUsingsCodeAction(Func<CancellationToken, Task<Document>> createChangedDocument)
-                : base(CSharpAnalyzersResources.Move_misplaced_using_directives, createChangedDocument)
+                : base(CSharpAnalyzersResources.Move_misplaced_using_directives, createChangedDocument, nameof(CSharpAnalyzersResources.Move_misplaced_using_directives))
```

Comment:
This change causing cleanup tests to fail. I still didn't figure out why, would be helpful if someone knows.

Reply:
In:

https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs#L150

`fixAllContext.CodeActionEquivalenceKey` is null, causing the following condition to be false:

https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs#L233-L238

Hence, no code actions are executed:

https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/Workspaces/Core/Portable/CodeFixes/FixAllOccurrences/BatchFixAllProvider.cs#L159

Reply:
I fixed that in e6b3238 and 4e54b71.

I used the first fix as similar thing is done in:

https://github.com/dotnet/roslyn/blob/f3462acb2744792c9f1654916e5d49a5aab67761/src/Features/Core/Portable/CodeFixes/CodeFixService.cs#L245-L247